### DISCUSSION TITLE
Organizations team role add multiple owner

### DISCUSSION THREAD

**Discussion by productdevbook** [supabase/supabase#486] [Questions] [answered]
More than one manager cannot be added at the moment. Only one person can be a manager. It would be nice to increase this number.

**Comment by thorwebdev** [comment]
Currently all members in a organisation are admin users. If you can't entrust this level of rights to your collaborators I'd recommend only sharing the Supabase API keys with your collaborators.

​We're planning for more fine grained access roles, this work is being tracked here: https://github.com/supabase/supabase/issues/3261 

​We'd appreciate if you could add your use case and needs there so we can take them into consideration.

**Reply by nkeat12** [reply]
Can only 1 'owner' invite other users? is there a way to grant the ability to add new developers/teamembers that join us to supabase?

**Reply by nkeat12** [reply]
@thorwebdev

**Reply by abhay187** [reply]
any update @thorwebdev

**Reply by thorwebdev** [reply]
You can find the different roles and their permission levels here: https://supabase.com/docs/guides/hosting/platform#manage-team-members

**Comment by thorwebdev** [comment, marked as answer]
You can find the different roles and their permission levels here: https://supabase.com/docs/guides/hosting/platform#manage-team-members
